onnxruntime/core/framework/onnxruntime_typeinfo.cc

```diff
@@ -10,6 +10,12 @@
 #include "core/framework/sparse_tensor.h"
 #include "core/graph/onnx_protobuf.h"
 #include "core/session/ort_apis.h"
+#include "core/framework/error_code_helper.h"
+
+#include "core/framework/tensor_type_and_shape.h"
+#include "../../winml/adapter/winml_adapter_map_type_info.h"
+#include "../../winml/adapter/winml_adapter_sequence_type_info.h"
```

Comment:
onnxruntime framework shouldn't include any headers from WinML. If they are needed, we should move them from WinML to onnxruntime folder.  This is for avoiding circular dependency.

Reply:
moved into onnxruntime/core/framework

---
In reply to: [370803657](https://github.com/microsoft/onnxruntime/pull/2891#discussion_r370803657) [](ancestors = 370803657)